scraper/visualize.py

```diff
@@ -1,5 +1,6 @@
 from typing import Iterable, Iterator
 import plotly.graph_objs as go
+from plotly import graph_objs
```

Comment:
Can you please remove this unused import?

Reply:
removed unused imports